src/commands/lpop.json

```diff
@@ -1,6 +1,6 @@
 {
     "LPOP": {
-        "summary": "Remove and get the first elements in a list",
+        "summary": "Remove and get one or multiple elements from the beginning of a list",
```

Comment:
i think the `one or multiple` part is excessive.
it's enough to say "Remove and get elements from the beginning of the list", or maybe "from the head of the list"

Reply:
come to think of it, i'm not sure the existing text is wrong.
it says "element**s**".
maybe all we need to do is add an `s` to the lpush and rpush json files.